docs/source/branch.rst

```diff
@@ -0,0 +1,218 @@
+proportion of the population that is recruited into the intervention program. We may want to run simulations on several
+different proportions including full recruitment and no recruitment, which would function as a baseline. We can easily
+do this with the following branches file.
+
+.. code-block:: yaml
+
+    branches:
+            - egg_intervention:
+                    recruitment:
+                        proportion: [0.0, 0.4, 0.8, 1.0]
+
+The ``branches`` block specifies changes to values found in the model specification YAML, exactly matching the blocks
+from that specification underneath.  Here, the YAML list [0.0, 0.4, 0.8, 1.0] dictates specific recruitment proportions
+to be simulated. Thus, you can expect four separate simulations to be run, one for each.
+
+.. warning::
+    Varying the time step, start or end time, or the population size of a simulation will make profiling very difficult
+    and runs the risk of breaking our output writing tools.
+
+
+Interaction with Uncertainty
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+As touched upon in the section on :ref:`combining draws and seeds<Combining Draws and Seeds>`, when multiple branch
+configurations would result in multiple simulations, the result is a simulation for every combination, or the cartesian
+product, of the parameters. Let's add draws to our previous branches file and figure out how many simulations it will
+result in.
+
+.. code-block:: yaml
+
+    input_draw_count: 100
+
+    branches:
+            - egg_intervention:
+                    recruitment:
+                        proportion: [0.0, 0.4, 0.8, 1.0]
+
+This branches file will result in 400 simulations, a set of 100 different draws for each recruitment proportion.
+
+Dual Parameter Variation
+^^^^^^^^^^^^^^^^^^^^^^^^
+Branches files really shine when you want to vary a lot of aspects of your model. Let's add another parameter to create
+scenarios along a new dimension. Say, for instance, we were also interested in the implementing the egg intervention
+by recruiting people only once they pass a certain age threshold. Provided components were available that can implement
+this,we could add a variety of starting ages to our branches file like so:
+
+.. code-block:: yaml
+
+    input_draw_count: 100
+
+    branches:
+            - egg_intervention:
+                    recruitment:
+                        proportion: [0.0, 0.4, 0.8, 1.0]
+                        age_start: [10.0, 25.0, 45.0, 65.0]
+
+This will result in scenarios encompassing every combination of recruitment proportion and starting age. Specifically,
+it will result in 100 separate simulations per combination, one for each randomly sampled draw. This means the total
+number of simulations is given by draws * proportions * starting age, or 1,600 altogether.
+
+.. note:: The top-level keys for controlling seeds and draws are the same across branches.
```

Comment:
Here again I find the terminology a little confusing - what does 'branches' mean in this note?